src/app/news/news-list.component.ts

```diff
@@ -82,6 +87,25 @@ export class NewsListComponent implements OnChanges {
       this.dialogsFormService.closeDialogsForm();
       this.dialogsLoadingService.stop();
     });
+    this.sendNewsNotifications(oldNews);
+  }
+
+  sendNewsNotifications(oldNews) {
+    const userId = this.userService.get()._id;
+    const link = '/community';
+    const notificationDoc = {
+      user: userId,
+      'message': 'Someone replyed to your community message.',
+      link,
+      'type': 'replyMessage',
+      'priority': 1,
+      'status': 'unread',
+      'time': this.couchService.datePlaceholder,
+      userPlanetCode: oldNews.messagePlanetCode
+    };
+    return this.couchService.findAll( 'notifications', ).pipe(
```

Comment:
The find needs a query, otherwise the notification will not be sent once there is any notifications in the system.